addons/l10n_in/views/account_move_line_views.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_move_line_tree_hsn_l10n_in" model="ir.ui.view">
+        <field name="name">account.move.line.tree.l10n_in</field>
+        <field name="model">account.move.line</field>
+        <field name="arch" type="xml">
+            <tree string="Products" editable="top" create="0">
+                <!-- Others fields -->
```

Comment:
Is this on purpose to have a view only for showing the HSN for the actionable errors widget?  I see things in there like many2one_barcode which maybe does not immediately make sense.

Reply:
> Is this on purpose to have a view only for showing the HSN for the actionable errors widget?

@jco-odoo, Yes.

Reply:
I think you can drastically simplify it then.  No need to change account, ...

Reply:
> I think you can drastically simplify it then. No need to change account, ...

so here we are keeping
Product - Description - HSN